tests/functional/includes/date.mock.js

```diff
@@ -0,0 +1,14 @@
+// Highjack the Date object so that our tests don't suddenly start failing...
+
+var __Date__ = Date;
+
+// February 13, 2015
+var STATIC_DATE_MS = 1423852666565;
+
+Date = function (value) {
+    return new __Date__(value !== undefined ? value : STATIC_DATE_MS);
+};
+
+Date.now = function () {
```

Comment:
`IntlRelativeFormat` takes a user-specified date value and compares it with `Date.now()` on every `format()` call, so to get the second date value it will hit this mocked method.